autogpt_platform/backend/backend/data/block.py

```diff
@@ -230,6 +230,7 @@ def __init__(
         self.disabled = disabled
         self.static_output = static_output
         self.block_type = block_type
+        self.execution_stats = {}
```

Comment:
Adding this for adding a dynamic persisted stats for blocks. This is persisted in NodeExecutions table